File: Array/medium/pb_169/code.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count=0
        element=-1
        for i in nums:
            if count == 0: 
                count = 1
                element=i
            elif i==element: count+=1
            else: count-=1
        return element        

File: Array/medium/pb_169/code.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int cnt = 0, el = -1;
        for(int i=0; i<nums.size();i++){
            if(cnt == 0){
                cnt = 1; 
                el=nums[i];
            }
            else if (nums[i] == el) cnt+=1;
            else cnt-=1;
        }
        return el;
    }
};